dev/js/actions/commons/login.js

```diff
@@ -0,0 +1,49 @@
+import {
+  SEND_USER_LOGIN_CREDENTIALS,
+  USER_LOGIN_FAIL,
+  USER_LOGIN_SUCCESS,
+} from '../../constants/types';
+import { API_POST } from '../../api/api.js';
+
+export const userLogin = (data, dispatch, cb) => {
+  dispatch({
+    type: SEND_USER_LOGIN_CREDENTIALS,
+    payload: {
+      loading: true,
+    },
+  });
+  API_POST('https://getkeel.com/api/v1/user/login', {
+    email: data.email,
+    password: data.password,
+  })
+    .then(function (response) {
+      if (response && response.status == 1) {
+        dispatch({
+          type: SEND_USER_LOGIN_CREDENTIALS,
+          payload: {
+            loading: false,
+          },
+        });
+        if (cb) cb(null, response);
+      }
+    })
+    .then(function () {
```

Comment:
there is no need for this ".then" you can do the same work in above then statement, you can club both dispatch in above ".then"